koocook_core/models/nutrition.py

```diff
@@ -28,6 +21,19 @@ class MetaIngredient(models.Model):
         blank=True,
     )
 
+    def save(self, *args, **kwargs):
+        if self.nutrient:
+            pass
+        else:
+            try:
+                from koocook_core.support.scripts import get_nutrients
+            except ModuleNotFoundError:
+                from koocook_core.management.commands._add_path import add_datatrans
+                add_datatrans()
+                from koocook_core.support.scripts import get_nutrients
```

Comment:
Why do you import inside the function? Does it help with circular dependencies or something?

Reply:
Prevents `datatrans` causing problems when not used